GradeServer.c

```diff
@@ -1,146 +1,216 @@
-#include <GradeServer.h>
-#include <ServerThread.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <unistd.h>
+#include <pthread.h>
+#include <stdlib.h>
 
-int sockfd;
+#define NUM_THREADS 5
+#define MAX_TASKS 1000
 
 
-int loadUsersFromFile(User * users, const char* filename, int user_type) {
```

Comment:
Why did you delete this function?

Reply:
That's what I told you that I deleted and needed to add back, but my brain was on fire alreay